monai/transforms/spatial/array.py

```diff
@@ -560,6 +573,19 @@ def __call__(
         out_d = self.pixdim[:sr]
         if out_d.size < sr:
             out_d = np.append(out_d, [1.0] * (sr - out_d.size))
+        orig_d = affine_to_spacing(affine_, sr, out_d.dtype)
+        for idx, (_d, mn, mx) in enumerate(
+            zip_longest(orig_d, self.min_pixdim[:sr], self.max_pixdim[:sr], fillvalue=np.nan)
+        ):
+            target = out_d[idx]
+            if np.isnan(mn) and np.isnan(mx):
+                continue
+            elif not np.isnan(mn) and np.isnan(mx):
+                out_d[idx] = _d if _d > mn else target
```

Comment:
I think in this case the user should set `max_pidxim` to be the same as the target `pixdim`. also `target is not necessarily larger than `mn` in the current implementation... this could give more flexibility though I don't know if there's a practical use case for it.

Reply:
What mean "`target` is not necessarily larger than `mn` in the current implementation"? `target` is not optional, why could it be smaller than `min_pixdim` if users set them both? And it seems in the current implementation, users must both set `min_pixdim` and `max_pixdim` to make sure the `min_pixdim` <  `target_pixdim`  < `max_pixdim`. But if users only set one of them and want the `min_pixdim` <  `target_pixdim`  <= `target_pixdim`, it seems it will return some wrong results unless giving warnings when only one of them is set.

Reply:
Currently the transform converts image in a source pixdim to a target pixdim. When the source pixdim is in the range of [min_pixdim, max_pixdim], the transform doesn't do the convert. in my understanding, min max are defined for the source pixdim, target is the output pixdim.

Reply:
thanks, I think you're right, using the target pixdim as the default min/max is a better default. for the special cases, the user can still set `max=np.inf` and `min=0.0` if they are really needed.